cypress/e2e/service-worker.test.ts

```diff
@@ -53,8 +53,11 @@ describe('Service Worker', () => {
   it('installs a ServiceWorker', () => {
     cy.visit('/', { serviceWorker: true })
       .get('#swap-page')
-      .wait('@NotInstalled', { timeout: 20000 })
-      .window({ timeout: 20000 })
+      .wait('@NotInstalled', {
+        // This is a large file, and takes some time to load.
```

Comment:
Does the increased timeout means that we're having downgraded performance when it comes to loading a service worker now?

Reply:
See https://github.com/Uniswap/interface/pull/6459#discussion_r1180898182